src/App.tsx

```diff
@@ -8,7 +8,26 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loader, setLoader] = useState(true);
```

Comment:
```suggestion
  const [loading, setLoading] = useState(true);
```
suggest better naming